packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -44,6 +40,7 @@ export default class DedicatedServerDashboard {
   }
 
   $onInit() {
+    this.$scope.$state = this.$state;
```

Comment:
**nitpick:** What is the need to attach `$state` to `$scope`. This is a bad idea.